<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frame</class>
 <widget class="QFrame" name="Frame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>403</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>340</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QFrame" name="frm_dynamics">
     <property name="minimumSize">
      <size>
       <width>791</width>
       <height>330</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #f0f0f0</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="minimumSize">
           <size>
            <width>581</width>
            <height>320</height>
           </size>
          </property>
          <property name="title">
           <string>Levels</string>
          </property>
          <widget class="QSlider" name="sld_x_0">
           <property name="geometry">
            <rect>
             <x>60</x>
             <y>40</y>
             <width>22</width>
             <height>200</height>
            </rect>
           </property>
           <property name="minimum">
            <number>-100</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="value">
            <number>0</number>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksAbove</enum>
           </property>
           <property name="tickInterval">
            <number>10</number>
           </property>
          </widget>
          <widget class="QSlider" name="sld_y_1">
           <property name="geometry">
            <rect>
             <x>150</x>
             <y>40</y>
             <width>22</width>
             <height>200</height>
            </rect>
           </property>
           <property name="minimum">
            <number>-100</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="value">
            <number>0</number>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksAbove</enum>
           </property>
           <property name="tickInterval">
            <number>10</number>
           </property>
          </widget>
          <widget class="QSlider" name="sld_z_2">
           <property name="geometry">
            <rect>
             <x>240</x>
             <y>40</y>
             <width>22</width>
             <height>200</height>
            </rect>
           </property>
           <property name="minimum">
            <number>-100</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="value">
            <number>0</number>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksAbove</enum>
           </property>
           <property name="tickInterval">
            <number>10</number>
           </property>
          </widget>
          <widget class="QSlider" name="sld_roll_3">
           <property name="geometry">
            <rect>
             <x>330</x>
             <y>40</y>
             <width>22</width>
             <height>200</height>
            </rect>
           </property>
           <property name="minimum">
            <number>-100</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="value">
            <number>0</number>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksAbove</enum>
           </property>
           <property name="tickInterval">
            <number>10</number>
           </property>
          </widget>
          <widget class="QSlider" name="sld_pitch_4">
           <property name="geometry">
            <rect>
             <x>420</x>
             <y>40</y>
             <width>22</width>
             <height>200</height>
            </rect>
           </property>
           <property name="minimum">
            <number>-100</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="value">
            <number>0</number>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksAbove</enum>
           </property>
           <property name="tickInterval">
            <number>10</number>
           </property>
          </widget>
          <widget class="QSlider" name="sld_yaw_5">
           <property name="geometry">
            <rect>
             <x>510</x>
             <y>40</y>
             <width>22</width>
             <height>200</height>
            </rect>
           </property>
           <property name="minimum">
            <number>-100</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="value">
            <number>0</number>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksAbove</enum>
           </property>
           <property name="tickInterval">
            <number>10</number>
           </property>
          </widget>
          <widget class="QLabel" name="label_8">
           <property name="geometry">
            <rect>
             <x>61</x>
             <y>269</y>
             <width>21</width>
             <height>20</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>X</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
          <widget class="QLabel" name="label_9">
           <property name="geometry">
            <rect>
             <x>150</x>
             <y>270</y>
             <width>21</width>
             <height>16</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Y</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
          <widget class="QLabel" name="label_10">
           <property name="geometry">
            <rect>
             <x>240</x>
             <y>270</y>
             <width>21</width>
             <height>16</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Z</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
          <widget class="QLabel" name="sld_roll">
           <property name="geometry">
            <rect>
             <x>320</x>
             <y>270</y>
             <width>41</width>
             <height>16</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Roll</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
          <widget class="QLabel" name="sld_pitch">
           <property name="geometry">
            <rect>
             <x>410</x>
             <y>270</y>
             <width>41</width>
             <height>16</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Pitch</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
          <widget class="QLabel" name="sld_yaw">
           <property name="geometry">
            <rect>
             <x>500</x>
             <y>270</y>
             <width>41</width>
             <height>16</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Yaw</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
          <widget class="QLabel" name="label">
           <property name="geometry">
            <rect>
             <x>19</x>
             <y>39</y>
             <width>31</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>100%</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_2">
           <property name="geometry">
            <rect>
             <x>16</x>
             <y>228</y>
             <width>41</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>-100%</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_3">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>134</y>
             <width>16</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="geometry">
            <rect>
             <x>580</x>
             <y>30</y>
             <width>171</width>
             <height>261</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>260</height>
            </size>
           </property>
           <property name="title">
            <string/>
           </property>
           <widget class="QPushButton" name="btn_mid_pos">
            <property name="geometry">
             <rect>
              <x>40</x>
              <y>30</y>
              <width>90</width>
              <height>23</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>90</width>
              <height>23</height>
             </size>
            </property>
            <property name="text">
             <string>Mid Pos</string>
            </property>
           </widget>
           <widget class="QFrame" name="animate_frame">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>80</y>
              <width>161</width>
              <height>171</height>
             </rect>
            </property>
            <property name="frameShape">
             <enum>QFrame::Box</enum>
            </property>
            <widget class="QPushButton" name="btn_animate">
             <property name="geometry">
              <rect>
               <x>40</x>
               <y>30</y>
               <width>75</width>
               <height>23</height>
              </rect>
             </property>
             <property name="text">
              <string>Animate</string>
             </property>
            </widget>
            <widget class="QLabel" name="label_4">
             <property name="geometry">
              <rect>
               <x>30</x>
               <y>70</y>
               <width>41</width>
               <height>16</height>
              </rect>
             </property>
             <property name="text">
              <string>steps</string>
             </property>
            </widget>
            <widget class="QLineEdit" name="txt_steps">
             <property name="geometry">
              <rect>
               <x>80</x>
               <y>70</y>
               <width>31</width>
               <height>20</height>
              </rect>
             </property>
             <property name="text">
              <string>40</string>
             </property>
            </widget>
            <widget class="QLineEdit" name="txt_cycles">
             <property name="geometry">
              <rect>
               <x>80</x>
               <y>130</y>
               <width>31</width>
               <height>20</height>
              </rect>
             </property>
             <property name="text">
              <string>1</string>
             </property>
            </widget>
            <widget class="QLabel" name="label_5">
             <property name="geometry">
              <rect>
               <x>30</x>
               <y>130</y>
               <width>41</width>
               <height>16</height>
              </rect>
             </property>
             <property name="text">
              <string>cycles</string>
             </property>
            </widget>
            <widget class="QLineEdit" name="txt_interval">
             <property name="geometry">
              <rect>
               <x>80</x>
               <y>100</y>
               <width>31</width>
               <height>20</height>
              </rect>
             </property>
             <property name="text">
              <string>100</string>
             </property>
            </widget>
            <widget class="QLabel" name="label_6">
             <property name="geometry">
              <rect>
               <x>30</x>
               <y>100</y>
               <width>41</width>
               <height>16</height>
              </rect>
             </property>
             <property name="text">
              <string>interval</string>
             </property>
            </widget>
            <widget class="QLabel" name="label_7">
             <property name="geometry">
              <rect>
               <x>120</x>
               <y>100</y>
               <width>21</width>
               <height>16</height>
              </rect>
             </property>
             <property name="text">
              <string>ms</string>
             </property>
            </widget>
            <widget class="QLabel" name="label_11">
             <property name="geometry">
              <rect>
               <x>120</x>
               <y>70</y>
               <width>31</width>
               <height>16</height>
              </rect>
             </property>
             <property name="text">
              <string>mod 4</string>
             </property>
            </widget>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
